cmake/serialConfigVersion.cmake

```diff
@@ -0,0 +1,21 @@
+set(PACKAGE_VERSION "1.2.1")
```

Comment:
Currently with Catkin, any newer version of this package is considered compatible.

With this one, the package is compatible:
1. if the requested version number is equal to the installed one
2. if the requested version number is inferior to the installed one *AND* the major version number matches (for "1.2.1", the major version number is "1")

So, if "0.9" is requested and "1.2.1" is installed => mismatch
If "1.1" is requested and "1.2.1" is installed => match

Relying on the major version number for compatibility seems to me like a good idea, and enables breaking changes to be introduced in order to keep this package alive [relates to #102 for instance].